newIDE/app/src/MainFrame/Onboarding/OnboardingDialog.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import userflow from 'userflow.js';
+import Dialog from '../../UI/Dialog';
+import FlatButton from '../../UI/FlatButton';
+import optionalRequire from '../../Utils/OptionalRequire';
+import { getProgramOpeningCount } from '../../Utils/Analytics/LocalStats';
+import { isMobile } from '../../Utils/Platform';
+import Window from '../../Utils/Window';
+import { loadPreferencesFromLocalStorage } from '../Preferences/PreferencesProvider';
+import { getUserUUID } from '../../Utils/Analytics/UserUUID';
+import RaisedButton from '../../UI/RaisedButton';
+import { Column, Line } from '../../UI/Grid';
+import { MarkdownText } from '../../UI/MarkdownText';
+
+const electron = optionalRequire('electron');
+const isDev = Window.isDev();
+export let isUserflowRunning = false;
+
+const onboardingText = `
+In 5 minutes, you will have:
+  - Created a game
+  - Learned the fundamentals of GDevelop
+
+(🇬🇧 The tour is only available in English)  
+We highly recommend it!
+`;
+
+const OnboardingDialog = () => {
+  const [open, setOpen] = React.useState(false);
+  const [loading, setLoading] = React.useState(false);
+
+  // Initialise userflow when modal opens.
+  React.useEffect(() => {
+    if (isDev) {
+      userflow.init('ct_y5qogyfo6zbahjejcbo3dybnta');
+    } else {
+      userflow.init('ct_paaz6o2t2bhlrlyi7a3toojn7e');
+    }
+    userflow.on(
+      // Undocumented legacy userflow event that is fired
+      // "when a flow either becomes active or removed"
+      // (tip given by a tech member of Userflow - it shouldn't be removed
+      // in the near future given the fact that some of their users still use it).
+      'flowvisibilitychange',
+      isRunning => {
+        isUserflowRunning = isRunning;
+        if (isRunning) {
+          setLoading(false);
+          setOpen(false);
+        }
+      }
+    );
+  }, []);
+
+  const startUserflow = React.useCallback(async () => {
+    setLoading(true);
+    const userPreferences = loadPreferencesFromLocalStorage();
+    const appLanguage = userPreferences ? userPreferences.language : undefined;
+    await userflow.identify(getUserUUID(), { language: appLanguage });
+    await userflow.start('b1611206-2fae-41ac-b08c-0f8ad72d8c39');
```

Comment:
Do we need to catch and set loading to false to give the user a chance to retry?

Reply:
I'll keep it really simple then, re-enable the button and that's it.
Because we don't have much control over such an error.